team_klm.js

```diff
@@ -30,4 +19,27 @@ const url = "https://flyapi.onrender.com";
   klm.append(firstP);
   klm.append(secondP);
   klm.append(thirdP);
-})();
+}
+
+async function getAirlineDataList() {
+  const url = "https://flyapi.onrender.com";
+  const lisnyc = fetch(`${url}/lis_nyc/klm`);
+  const tlvber = fetch(`${url}/tlv_ber/klm`);
+  const arnlhr = fetch(`${url}/arn_lhr/klm`);
+
+  const response = await Promise.all([lisnyc, tlvber, arnlhr]);
+  const dataList = await Promise.all(response.map(async (entry) => await entry.json()));
+
+  return dataList;
+}
+
+function setLogo() {
+  const img = document.createElement("img");
+  img.src = "https://logo.clearbit.com/klm.com";
+  img.width = "100";
+  return img;
+}
+
+window.addEventListener("load", () => {
```

Comment:
Why onload? if this is called at the end of the body, I'm not sure you need to do this.